### DISCUSSION TITLE
ERROR Deployer not found: git but can on Ubuntu VM

### DISCUSSION THREAD

**Discussion by Mizoreeeee** [hexojs/hexo#5485] [Question / Answer] [answered]
ERROR Deployer not found: git
I have already installed git and hexo-deployer-git and added to PATH, I could use git and hexo directly by powershell in windows terminal.
And before I deploy on windows, I deploied hexo on Ubuntu 22.04LTS on a VM.
I deploied on Ubuntu successfully and ran as expected, could upload to my ali ecs.
Where is the problem?
![{56449AF6-BA47-46ed-9872-BB6F2B6D58D3}](https://github.com/hexojs/hexo/assets/77111118/e3ca005b-9cec-4cd7-8b8d-de2a3a07ae21)
![{AFC2A33F-A148-4d2d-B2C9-CD204EEF31C9}](https://github.com/hexojs/hexo/assets/77111118/92cdb6f7-9888-47f4-aeac-dffa9c2fe702)

**Comment by Mizoreeeee** [comment, marked as answer]
I solve the problem. I am not familiar with node.js and npm. It seems that I installed hexo-deployer-git gloally but not in the working directory of hexo. Perheps when I installed it, my pwd is not the working directory of hexo or `hexo init` will not automaticly install it.

**Comment by uiolee** [comment]
`hexo-deployer-git` should be installed in your project directory instead of installed globally.

Use `npm install <package>` instead of `npm install -g <package>`
